app/views/recipes/shopping_list.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="shopping_list_container">
+  <h1>Shopping List</h1>
+  <h4>Amount of food items to buy: <%= @comparison_food.count %>  Total value of food needed: 
+    <% sum = 0 %>
+    <% @comparison_food.each do |cf| %>
+      <% sum += cf.quantity * cf.price %>
+    <% end %>
+    <%= sum %>
+  <h4>
+  
+  <br />
+  <% if @comparison_food.count == 0 %>
+    <p>You have all the ingredients</p>
+  <% else %>
+    <table>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+      <% @comparison_food.each do |f| %>
+      <tr>
+        <td><%= f.name %></td>
+        <td><%= f.quantity %> <%= f.measurement_unit %></td>
+        <td><%= f.price %></td>
```

Comment:
- [ ] Please note that the price here shouldn't be the price of a unit of that item but the price of the quantity of that item. Assuming rice is 1000 per unit and the shopping list has 3 units of rice, the price here should be 3000. Kindly fix that, please.
![image](https://user-images.githubusercontent.com/94551484/221010886-05531ac4-a451-4166-ac6f-fef1a603194a.png)